eng/common/build.sh

```diff
@@ -87,6 +87,8 @@ prepare_machine=false
 verbosity='minimal'
 runtime_source_feed=''
 runtime_source_feed_key=''
+host_arch=''
+target_os=''
```

Comment:
Its to avoid 
```
/home/scott/github/runtimelab/eng/common/build.sh: line 245: host_arch: unbound variable
```
`${host_arch-''}` might be another way to solve it, or make it a mandatory parameter.

Reply:
But the only uses of `host_arch` that I see look to be new (added in this change). My suggestion is delete them - or are they needed for something?